py/picca/delta_extraction/expected_flux.py

```diff
@@ -350,8 +350,11 @@ def hdu_cont(self, results):
             self.get_mean_cont(Forest.log_lambda_rest_frame_grid),
             self.get_mean_cont_weight(Forest.log_lambda_rest_frame_grid),
         ],
-                      names=['loglam_rest', 'mean_cont', 'weight'],
+                      names=['LOGLAM_REST', 'MEAN_CONT', 'WEIGHT'],
+                      units=['log(Angstrom)', '10**-17 erg/(s cm2 Angstrom)', ''],
```

Comment:
I'm thinking we should load this from the `Data` class or the `Forest` class, as different surveys might use different units for the flux

Reply:
I have added this. Only for fluxes. At first I did it also for wavelength, but I think it makes no sense to run picca in other units (for example BAL lines are defined in Angstroms, so we will need to write an unit converter)